src/ripple/rpc/impl/ServerHandlerImp.cpp

```diff
@@ -598,11 +610,40 @@ ServerHandlerImp::processRequest (Port const& port,
             continue;
         }
 
+        auto apiVersion = RPC::APIVersionIfUnspecified;
+        if (jsonRPC.isMember(jss::params) &&
+            jsonRPC[jss::params].isArray() &&
+            jsonRPC[jss::params].size() > 0 &&
+            jsonRPC[jss::params][Json::UInt(0)].isObject())
```

Comment:
`jsonRPC[jss::params][0u]....` might be little more concise (in general for indexing of json arrays with static values)

Reply:
Thanks! I changed the code.